torchvision/models/quantization/googlenet.py

```diff
@@ -77,41 +85,41 @@ def googlenet(pretrained=False, progress=True, quantize=False, **kwargs):
 
 class QuantizableBasicConv2d(BasicConv2d):
 
-    def __init__(self, *args, **kwargs):
+    def __init__(self, *args: Any, **kwargs: Any,) -> None:
```

Comment:
If it' on a single line, let's remove the last comma

Reply:
Cool! I think I have done it.